sway/desktop/render.c

```diff
@@ -1497,6 +1704,12 @@ void output_render(struct sway_output *output, struct timespec *when,
 		render_layer_toplevel(output, damage,
 			&output->layers[ZWLR_LAYER_SHELL_V1_LAYER_BOTTOM]);
 
+		// TODO: Also check if there are any windows that would benefit from this
+		bool blur_enabled = config->blur_enabled && should_parameters_blur();
```

Comment:
this omits the case of blur being disabled globally but enabled for specific windows

Reply:
thats probably what your comment was addressing, but just wanted to comment to bring some more visibility here

Reply:
> this omits the case of blur being disabled globally but enabled for specific windows

Yeah. Loop through the windows and check if there are any windows that would benefit (tiled or floating with xray)